assets/css/components/_form-authentication.scss

```diff
@@ -0,0 +1,6 @@
+.form-authentication {
+  width: 100%;
+  max-width: rem(400px);
```

Comment:
As a best practice, components should not specific sizing. Build them always to fit 100% of the screen. This way, components can be placed anywhere.

The horizontal centring of the form should be achieved via the layout styles. This would allow all content is centred on all pages using this layout. For instance, `<main/>` tag could be used and styled to be centred. Then the form would be placed inside `<main/>`.

Reply:
Yes, I refactor this layout and centering elements by the layout styles instead.